packages/Modal/__tests__/Modal.spec.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import Modal from '../Modal'
+
+describe('Modal', () => {
```

Comment:
Some tests could be added:

- Check if confirm and cancel buttons appear when using Dialogue Modal 
- Check if confirm and close buttons appear when using Content Modal
- Ensure heading appears using level-3 styles (using `level={h3} tag="div"` props)